docs/source/Introduction.rst

```diff
@@ -13,7 +13,9 @@ Setup
 
 First install the following dependencies (`pip install` is your friend):
 
-* ``django``
+* ``django`` 
+   Python 2.7 only supports django 1.11 please use:
```

Comment:
This is not rendering well:
https://github.com/omps/avocado-server/blob/f4da01b618684b9525cffff76a52447583836b89/docs/source/Introduction.rst

Could you please improve the situation a bit?

Reply:
which part?

Reply:
The sentence "Python 2.7..." is being considered continuation line of the previous bullet item. Maybe the best solution would be: 

```
* ``django`` (with Python 2.7, use ``django==1.11``) 
```

Reply:
Actually the whole paragraph is doomed to be outdated as the `requirements.txt` is the file that should be used to get dependencies. Currently there are hardcoded dependencies as it's being used by Travis, but we could create `requirements-travis.txt` and use `>=` and `<=` to specify the ranges.

@omps do you feel like doing this or should we change it ourselves? (basically it's about changing the docs to use `pip install -r requirements.txt`, `cp requirements.txt requirements-travis.txt`, changing `.travis.yml` to use `requirements-travis.txt` and then modifying `requirements.txt` to use ranges with `python2` dependency (see https://github.com/avocado-framework/avocado/blob/master/requirements.txt for conditional deps)